src/lib/settings.ts

```diff
@@ -29,4 +29,8 @@ export class SettingsManager {
     public static getGraphEngineJvmArgs(): string {
         return vscode.workspace.getConfiguration('codeAnalyzer.graphEngine').get('jvmArgs');
     }
+
+    public static getAnalyzeOnSave(): boolean {
+        return vscode.workspace.getConfiguration('codeAnalyzer.analyze-on-save').get('enabled');
```

Comment:
You'll need to add a new property for this in the `contributes.configuration.properties` section of `package.json`, I believe. You'll also need to clear the property's label and help text with Teresa.

Reply:
I sent a separate PR for that - https://github.com/forcedotcom/sfdx-code-analyzer-vscode/pull/46 and got it merged. I didn't get a review from @teresa-allen-sfdc. I will follow up on that.